skmultilearn/model_selection/iterative_stratification.py

```diff
@@ -321,7 +327,11 @@ def _distribute_negative_evidence(self, rows_used, folds):
             row = available_samples.pop()
             rows_used[row] = True
             samples_left -= 1
-            fold_selected = np.random.choice(np.where(self.desired_samples_per_fold > 0)[0], 1)[0]
+            fold_selected = self._
+            
+            
+            
+            state.choice(np.where(self.desired_samples_per_fold > 0)[0], 1)[0]
```

Comment:
gotta fix this

Reply:
Yes, thank you! Not sure how that leaked into the PR